js/bleutrade.js

```diff
@@ -203,4 +205,19 @@ module.exports = class bleutrade extends bittrex {
             throw new ExchangeError (this.id + ' publicGetOrderbook() returneded no result ' + this.json (response));
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
+
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let response = await this.accountGetOrderhistory (params);
+        let trades = this.parseTrades (response['result'], market, since, limit);
+        for (let i = 0; i < trades.length; i++) {
+            let trade = trades[i];
+            trade['order'] = id;
```

Comment:
This won't work in PHP, unfortunately, as it copies arrays by value, not by reference. Will rewrite it shortly.

Reply:
> Would Object.assign(...) work? Or is there a CCXT method for copying objects?

Not really... The issue here is explained in the following way:

If you do the following in JS:

```JavaScript
let trades = [ ... ]; // an array of trades
for (let i = 0; i < trades.length; i++) {
    let trade = trades[i]; // js creates a reference to trades[i] called trade
    trade['order'] = id; // the order property will be set on trades[i]['order']
}
```

In PHP, however, when you do:

```PHP
let trades = array ( ... ); // an array of trades
for ($i = 0; $i < count($trades); $i++) {
    $trade = $trades[i]; // PHP creates a copy of trades[i] and calls it a trade
    $trade['order'] = $id; // the order property will be set in the copy, not in trades[i]['order']
}
```

So, I will simply rewrite those lines to something like this:

```JavaScript
let result = [];
let trades = this.parseTrades (...);
for (let i = 0; i < trades.length; i++) {
    // this creates a copy in php
    let trade = this.extend (trades[i], {
        'order': id,
    });
    if ((trade['symbol'] === undefined) && (market !== undefined)) {
        trade['symbol'] = market['symbol'];
    }
    // the copy is pushed into resulting array
    result.push (trade);
}
return result; // the resulting array of copies is returned
```